python_ta/reporters/templates/stylesheet.css

```diff
@@ -271,6 +271,10 @@ section {
 .code-errors {
 }
 
+code {
+  padding: 0em 0.35em;
```

Comment:
Add a background colour to make this stand out a bit more, with some rounded corners. See, for example, https://picocss.com/docs and https://github.com/picocss/pico/blob/main/css/pico.css.